data/transition-sites/phe_chimat.yml

```diff
@@ -1,10 +1,7 @@
 ---
 site: phe_chimat
 whitehall_slug: public-health-england
-homepage: https://www.gov.uk/government/organisations/public-health-england
-homepage_furl: www.gov.uk/phe
```

Comment:
The [`homepage_furl` should redirect to the `homepage`](https://github.com/alphagov/transition-config#site-configuration), so should be removed in this case.